app/controllers/api/v2/blocks_controller.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class Api::V2::BlocksController < Api::V1::BlocksController
+  def index
+    @blocks = load_blocks
+    render json: @blocks, each_serializer: REST::BlockSerializer
+  end
+
+  def load_blocks
+    paginated_blocks.to_a
```

Comment:
What's happening here?

Reply:
It inherits from `Api::V1::BlocksController` and uses exactly the same logic to select which blocks to show (`Api::V1::BlocksController#paginated_blocks`), except it shows the blocks themselves and not the accounts.